### PR TITLE
fix(cactus-example-supply-chain-app): mitigate CVE-2022-24434 and CVE-2022-24999

### LINKED ISSUE #2041: fix(security): vulnerabilities found in cactus-example-supply-chain-app
List of vulnerabilities found in cactus-example-supply-chain-app image during Azure Container scan.

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/ZONDER~1.CAL/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/ZONDER~1.CAL/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

</head>

<body link="#0563C1" vlink="#954F72">

VULNERABILITY ID | PACKAGE NAME | SEVERITY
-- | -- | --
CVE-2021-3807 | ansi-regex | HIGH
CVE-2021-3807 | ansi-regex | HIGH
CVE-2021-43138 | async | HIGH
CVE-2022-22143 | convict | HIGH
CVE-2020-8203 | lodash | HIGH
CVE-2021-23337 | lodash | HIGH
CVE-2022-24771 | node-forge | HIGH
CVE-2022-24772 | node-forge | HIGH
CVE-2021-23358 | underscore | HIGH

</body>

</html>

### PR DESCRIPTION
**Pull request by aldousalvarez**
Fixes #2041

Signed-off-by: aldousalvarez <aldousss.alvarez@gmail.com>

### CODE REVIEW

**Comment by aldousalvarez** [comment]
Hello Peter, Most of the vulnerabilities are now fixed in cactus-example-supply-chain-app but there are still some that is not yet fixed as you can see [here](https://github.com/hyperledger/cactus/actions/runs/3728650724/jobs/6323849435
). The only vulnerability (CVE-2022-2421) is still a vulnerability because the latest version of the package that is being used is still the affected version. The changes committed on this PR will fix the 3 out of the 4  remaining vulnerabilities (CVE-2022-24434, CVE-2022-24999, and CVE-2022-24999) once the changes are applied and the new version is released and the packages are updated just like the v1.1 release.

File: .github/containerscan/allowedlist.yaml [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
+    - CVE-2022-37734
+    - CVE-2022-25857
 
-    -CVE-2022-37734
-    -CVE-2022-25857
+    #cactus-example-supply-chain-app
+    - CVE-2022-24434
+    - CVE-2022-24999 #express
+    - CVE-2022-24999 #qs
+    - CVE-2022-2421
```
Comment (petermetz) [inline .github/containerscan/allowedlist.yaml:11]: @aldousalvarez Let's just leave these out for now, as-in, revert the changes in allowedlist.yaml 'coz I think we can find some way to fix those vulnerabilities and in the meantime they shouldn't be masked unless absolutely necessary.
Other than this, it LGTM

**Review by petermetz — CHANGES_REQUESTED** [review]

**Review by izuru0 — APPROVED** [review]
LGTM

File: .github/containerscan/allowedlist.yaml (hunk 1 above)
Comment (aldousalvarez) [reply .github/containerscan/allowedlist.yaml]: Okay Thank you, noted on that one. Already fixed it.

**Review by petermetz — APPROVED** [review]
@aldousalvarez LGTM, thank you!

**Review by petermetz — CHANGES_REQUESTED** [review]
@aldousalvarez Sorry, my bad, I do need to ask that you put CVE IDs in the commit subject so that it's unique across the commit log. Other than that, we are good to go.

**Comment by aldousalvarez** [comment]
Hello @petermetz also updated the commit subject and PR title as from your requested changes on this one. Thank you.

**Review by petermetz — APPROVED** [review]
@aldousalvarez Great, thank you for the updates, LGTM!

**Pull request merged**